src/topology/vector_bundle.lean

```diff
@@ -78,6 +90,13 @@ namespace topological_vector_bundle
 
 variables {R F E}
 
+/-- Natural identification as `topological_vector_prebundle.trivialization`. -/
```

Comment:
natural identification of what?

Reply:
Of a trivialization, you want it to be specified in the comment even if it's obvious from the code?

Reply:
Yes, sorry, I wasn't clear, my comment meant that it should be explained in the docstring. Often, when looking for a lemma, I'll use autocompletion and then just look at the docstrings, which are usually much more readable than the code. A half-self-contained docstring is useless in these situations.